### PR TITLE
Push thehive v5

### PR DESCRIPTION
**Pull request by TOUFIKIzakarya**
Add a new version of thehive
related issue : [#55594](https://github.com/SekoiaLab/platform/issues/55594)

### CODE REVIEW

**Review by squioc — APPROVED** [review]
LGTM. I suggest adding the version number in the titles to to make a visual difference

File: TheHiveV5/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+[tool.poetry]
+name = "Automation Module for The Hive"
+description = ""
+version = "0.0"
+authors = []
+
+[tool.poetry.dependencies]
+python = ">=3.10,<3.12"
+sekoia-automation-sdk = "^1.8.1"
```
Comment (squioc) [inline TheHiveV5/pyproject.toml:9]:
```suggestion
sekoia-automation-sdk = "^1.13.0"
```

Comment (TOUFIKIzakarya) [reply TheHiveV5/pyproject.toml:9]: Ok yes thank you for this catch

**Review by squioc — APPROVED** [review]
LGTM

**Pull request merged**